### DISCUSSION TITLE
delete with SQLite_web

### DISCUSSION THREAD

**Discussion by calz64** [hassio-addons/addon-sqlite-web#179] [Q&A]
Buongiorno, 
I deleted some row in my database with SQLite_web, but i didn't see any change in Home Assistant. I tried to delete some state of 'shelly sensor power' because i had wrote manually some wrong value, and in the card 'energy usage' of Home Assistant i had a wrong statistic. Why? Do SQLite_web really delete the data on database of Home Assistant? or it use a virtual file.
Thanks, and sorry for my english

**Comment by frenck** [comment]
the energy dashboard doesn't use the states directly.

For more information on the data model of Home Assistant, visit the Home Assistant data portal:

<https://data.home-assistant.io>

The use of the model is out of scope for this add-on IMHO.

**Reply by calz64** [reply]
Thanks a lot, i think that sadly I'll keep energy dashboard with the wrong statistic. :)

**Reply by frenck** [reply]
In an upcoming release of Home Assistant, developer tools for managing statistics are expected.
